vue/dynamicforms/src/index.ts

```diff
@@ -54,6 +55,7 @@ export function createDynamicForms(options: DynamicFormsOptions = defaultOptions
 
       // import all global instances that we need for vuetify to work
       Object.entries(VuetifyComponents).map(([name, component]) => app.component(name, component));
+      app.component('df-app', DfApp);
```

Comment:
I still get this error in the console:
~/vue/dynamicforms/src/index.ts
  58:21  warning  Property name "df-app" is not PascalCase  vue/component-definition-name-casing

Reply:
To reproduce... npm run dev -> console output

Reply:
I already changed this once based on a comment, from dfApp -> df-app.

So should it be PascalCase or kebab-case?